doc/classes/BakedLightmap.xml

```diff
@@ -45,6 +49,7 @@
 			Lightmapping mode. See [enum BakeMode].
 		</member>
 		<member name="bake_propagation" type="float" setter="set_propagation" getter="get_propagation" default="1.0">
+			Defines how far the light will travel before it is no longer effective. The higher the number the farther the light will travel. For instance, if the value is set to 2 the light will go twice as far. If it is 0.5 it will go half as far.
```

Comment:
```suggestion
			Defines how far the light will travel before it is no longer effective. The higher the number, the farther the light will travel. For instance, if the value is set to 2, the light will go twice as far. If the value is set to 0.5, the light will only go half as far.
```

Reply:
Just updated the docs using all of the suggestions above.